diff --git a/node_modules/@cloudflare/speedtest/.DS_Store b/node_modules/@cloudflare/speedtest/.DS_Store
new file mode 100644
index 0000000..08f2cb1
Binary files /dev/null and b/node_modules/@cloudflare/speedtest/.DS_Store differ
diff --git a/node_modules/@cloudflare/speedtest/dist/speedtest.js b/node_modules/@cloudflare/speedtest/dist/speedtest.js
index 10bd295..251d04b 100644
--- a/node_modules/@cloudflare/speedtest/dist/speedtest.js
+++ b/node_modules/@cloudflare/speedtest/dist/speedtest.js
@@ -341,7 +341,7 @@ function _classPrivateMethodInitSpec(obj, privateSet) {
   privateSet.add(obj);
 }
 
-var REL_API_URL = 'https://speed.cloudflare.com';
+var REL_API_URL = import.meta.env.VITE_SPEEDTEST_REVERSE_URL || 'https://speed.cloudflare.com';
 var defaultConfig = {
   // Engine
   autoStart: true,
